prow/Makefile

```diff
@@ -1,16 +1,16 @@
 .phony: update-config update-plugins validate
 
 update-config:
-	kubectl create configmap config --from-file=config.yaml=config/config.yaml --dry-run -o yaml | kubectl replace configmap config -f -
+	kubectl create configmap config --from-file=config.yaml=config/config.yaml --dry-run=client -o yaml | kubectl replace configmap config -f -
 
 update-plugins:
-	kubectl create configmap plugins --from-file=plugins.yaml=config/plugins.yaml --dry-run -o yaml | kubectl replace configmap plugins -f -
+	kubectl create configmap plugins --from-file=plugins.yaml=config/plugins.yaml --dry-run=client -o yaml | kubectl replace configmap plugins -f -
 
 validate:
 	podman run --rm \
 		--volume "${PWD}:/workdir:ro,z" \
 		--entrypoint /checkconfig \
-		gcr.io/k8s-prow/checkconfig:v20191219-ecbeba384 \
+		gcr.io/k8s-prow/checkconfig:v20210409-985ef5e721 \
```

Comment:
how often this needs to change?

Reply:
It is not mandatory but good to be up to date with upstream. We can do it once a month or so. For example, until now we were using images from 2019 and it was still fine.